tests/models/phi/test_modeling_phi.py

```diff
@@ -360,6 +366,96 @@ def test_phi_sequence_classification_model_for_multi_label(self):
         result = model(input_ids, attention_mask=attention_mask, labels=sequence_labels)
         self.assertEqual(result.logits.shape, (self.model_tester.batch_size, self.model_tester.num_labels))
 
+    @parameterized.expand([("linear",), ("dynamic",)])
+    def test_model_rope_scaling_from_config(self, scaling_type):
```

Comment:
This one is not a new test, but rather a missing test. It is copy/paste from the same test in Falcon (Phi is mostly copied from Falcon)